Makefile

```diff
@@ -27,15 +27,20 @@ endif
 ##@ Default target
 all: gitops ## Install dependencies and build Gitops binary
 
+
+TEST_TO_RUN=./...
 ##@ Test
-unit-tests: dependencies  ## Run unit tests
+unit-tests: dependencies ## Run unit tests
+	which ginkgo || go install github.com/onsi/ginkgo/v2/ginkgo
```

Comment:
Another question on the Ginkgo check: is that worth pulling out somewhere, so that it only occurs in one place rather than four? That seems to make sense to me as a separate phony target, but there may well be more idiomatic Make-y ways to do it.

Reply:
The integration tests are gone, the acceptance tests are almost gone, and the `test` and `unit-tests` target do basically the same thing so I implemented one in terms of the other.

Reply:
Fair enough. That sounds like a good argument for it not being worth pulling out.